src/guides/v2.4/release-notes/commerce-2-4-1.md

```diff
@@ -683,27 +858,123 @@ We have fixed hundreds of issues in the Magento 2.4.1 core code.
+
+<!--- ENGCOM-7713-->
+
+*  Magento no longer throws an `Undefined class constant` error when an interceptor is generated. _Fix submitted by Vova Yatsyuk in pull request [28797](https://github.com/magento/magento2/pull/28797)_. [GitHub-28981](https://github.com/magento/magento2/issues/28981)
+
+<!--- ENGCOM-7523-->
+
+*  Form data now persists when Magento throws an integration exception when you save a integration using a name that is already in use. _Fix submitted by Aditya Yadav in pull request [26660](https://github.com/magento/magento2/pull/26660)_. [GitHub-28143](https://github.com/magento/magento2/issues/28143)
+
+<!--- ENGCOM-7756-->
+
+*  Magento no longer truncates `X-Forwarded-For` headers to 32 characters. _Fix submitted by Ihor Sviziev in pull request [27221](https://github.com/magento/magento2/pull/27221)_. [GitHub-28693](https://github.com/magento/magento2/issues/28693)
+
+<!--- ENGCOM-7820-->
+
+*  The logic that checks if a redirect is internal now works correctly in the Admin when using a custom Admin domain. Previously, problems with this logic resulted in many Admin redirects to the homepage of the default store. _Fix submitted by Vadim Malesh in pull request [29066](https://github.com/magento/magento2/pull/29066)_. [GitHub-28943](https://github.com/magento/magento2/issues/28943)
+
+<!--- ENGCOM-7758-->
+
+*  Problems with the `styles-old.less` file have been eliminated,  and linting no longer identifies errors. _Fix submitted by Tu Nguyen in pull request [28895](https://github.com/magento/magento2/pull/28895)_. [GitHub-24004](https://github.com/magento/magento2/issues/24004)
+
+<!--- ENGCOM-7781-->
+
+*  `NonComposerComponentRegistration.php` has been refactored.  _Fix submitted by Vitaliy Ryaboy in pull request [28975](https://github.com/magento/magento2/pull/28975)_. [GitHub-29308](https://github.com/magento/magento2/issues/29308)
+
+<!--- ENGCOM-7926-->
+
+*  `ResourceConnection.php` has been refactored to improve class readability. _Fix submitted by Lukasz Bajsarowicz in pull request [29341](https://github.com/magento/magento2/pull/29341)_. [GitHub-29389](https://github.com/magento/magento2/issues/29389)
+
+<!--- ENGCOM-7910-->
+
+*  The README file for the build-in web server has been updated to include all Elasticsearch parameters. _Fix submitted by Yevhenii Dumskyi in pull request [29300](https://github.com/magento/magento2/pull/29300)_. [GitHub-29299](https://github.com/magento/magento2/issues/29299)
+
+<!--- ENGCOM-7814-->
+
+*  All after plugins now return a value as expected. Previously, `Magento\CmsUrlRewrite\Plugin\Cms\Model\Store\View::aftersSave` did not return a value, and as a result, saving a store view resulted in an error. _Fix submitted by Pieter Hoste in pull request [29035](https://github.com/magento/magento2/pull/29035)_. [GitHub-29034](https://github.com/magento/magento2/issues/29034)
```

Comment:
I didn't check *all* after plugins, so this statement could be incorrect. I only fixed this one after plugin which gave us errors in one particular shop 🙂